personalize.module

```diff
@@ -808,8 +793,6 @@ function personalize_theme($existing, $type, $theme, $path) {
       'variables' => array(
         'variation_title' => t('Variation set'),
         'variation_count' => 0,
-        'report' => array(),
```

Comment:
Should remove this section from theme_personalize_wizard_variations_header as well then.